docs/developers/Building_Android.md

```diff
@@ -10,17 +10,17 @@ The following instructions apply to **v1.2 and later**. For earlier versions the
 2. JDK 11, not necessarily from Oracle
 3. Android command line tools or Android Studio for your OS: https://developer.android.com/studio/
 4. Android NDK version **r25c (25.2.9519653)**, there're multiple ways to obtain it:
-- - install with Android Studio
-- - install with `sdkmanager` command line tool
-- - download from https://developer.android.com/ndk/downloads
-- - download with Conan, see [#NDK and Conan](#ndk-and-conan)
-5.  (optional) Ninja: download from your package manager or from https://github.com/ninja-build/ninja/releases
+    - install with Android Studio
+    - install with `sdkmanager` command line tool
+    - download from https://developer.android.com/ndk/downloads
+    - download with Conan, see [#NDK and Conan](#ndk-and-conan)
+5. (optional) Ninja: download from your package manager or from https://github.com/ninja-build/ninja/releases
 
 ## Obtaining source code
 
 Clone https://github.com/vcmi/vcmi with submodules. Example for command line:
 
-```sh
+```
```

Comment:
Don't mind the change, but why?

Reply:
Doesn't add any value: no bash stuff to highlight